runtimes/peregrine/src/tests.rs

```diff
@@ -30,6 +39,61 @@ fn call_size() {
 	);
 }
 
+#[test]
+fn attestation_storage_sizes() {
+	let attestation_record = attestation::AttestationDetails::<Runtime>::max_encoded_len();
+	let delegation_record = BoundedVec::<
+		<Runtime as frame_system::Config>::Hash,
+		<Runtime as attestation::Config>::MaxDelegatedAttestations,
+	>::max_encoded_len()
+		/ (<Runtime as attestation::Config>::MaxDelegatedAttestations::get() as usize);
+	assert_eq!(
+		attestation_record + delegation_record,
+		MAX_ATTESTATION_BYTE_LENGTH as usize
+	)
+}
+
+#[test]
+fn did_storage_sizes() {
+	let did_size = did::did_details::DidDetails::<Runtime>::max_encoded_len();
+
+	// service endpoints and counter
+	let did_endpoint_size = did::service_endpoints::DidEndpoint::<Runtime>::max_encoded_len()
+		* (<Runtime as did::Config>::MaxNumberOfServicesPerDid::get() as usize)
+		+ u32::max_encoded_len();
+
+	assert_eq!(did_size + did_endpoint_size, MAX_DID_BYTE_LENGTH as usize)
+}
+
+#[test]
+fn did_lookup_storage_sizes() {
+	let did_connection_size =
+		pallet_did_lookup::ConnectionRecord::<
+			<Runtime as pallet_did_lookup::Config>::DidIdentifier,
+			<Runtime as frame_system::Config>::AccountId,
+			<<Runtime as pallet_did_lookup::Config>::Currency as Currency<
+				<Runtime as frame_system::Config>::AccountId,
+			>>::Balance,
+		>::max_encoded_len();
+
+	assert_eq!(did_connection_size, MAX_CONNECTION_BYTE_LENGTH as usize)
+}
+
+#[test]
+fn web3_name_storage_sizes() {
+	let owner_size = Web3NameOf::<Runtime>::max_encoded_len();
+	let name_size = Web3OwnershipOf::<Runtime>::max_encoded_len();
+
+	assert_eq!(owner_size + name_size, MAX_NAME_BYTE_LENGTH as usize)
+}
+type Indices = (<Runtime as frame_system::Config>::AccountId, BalanceOf<Runtime>, bool);
+
+#[test]
+fn indices_storage_sizes() {
```

Comment:
S: Move this into the test scope as the type is not needed elsewhere.
```suggestion

#[test]
fn indices_storage_sizes() {
type Indices = (<Runtime as frame_system::Config>::AccountId, BalanceOf<Runtime>, bool);
```

Reply:
3ac2a64741db0b64ef7ce77fd1fdebf56d3a4147